backend/app/api/routes/conversations.py

```
@@ -0,0 +1,487 @@
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail="Content item not found"
+        )
+    
+    # 查询对话
+    query = select(AIConversation).where(
+        AIConversation.content_item_id == content_id,
+        AIConversation.user_id == current_user.id
+    )
+    
+    if not include_inactive:
+        query = query.where(AIConversation.is_active == True)
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if AIConversation.is_active:` for truth checks

Replace with `AIConversation.is_active`

(E712)